package.json

```
@@ -104,7 +104,7 @@
   },
   "dependencies": {
     "@across-protocol/across-token": "^1.0.0",
-    "@across-protocol/constants": "^3.1.59",
+    "@across-protocol/constants": "^3.1.60",
```

Can we also include the newest version of constants here?
```suggestion
    "@across-protocol/constants": "^3.1.61",
```